bendowen/BEN_DOWEN.md

```diff
@@ -0,0 +1,74 @@
+
+At first I thought I would simply find all the winners myself. Simply look on Twitter, find someone saying something cool about Software Testing and then boom, award them Tester of The Day.
+
+A few days in, and I was thinking maybe this will last a week. But then something amazing happened. The awesome power of the Software Testing Community awoke. People started to nominate people to win Tester of the Day.
+
+The community were liking and following the Winners, congratulating them, celebrating them. Some of the early supporters like Chris Armstrong, Arleen Andrews and Marie Drake went on to nominate multiple winners.
+
+Momentum started to build, and I felt an obligation, and some considerable excitement, to keep going. So I built a site, while on a Summer Holiday at my Mum's Seaside Chalet, a precious thing in 2020.
+
+## Celebrating the Community
+
+At the time of writing, more Tester of The Day has been awarded more then 150 times since it was started on the 17th of August 2020. With only a handful of people winning more then once.
+
+I am very humbled by the joy and excitement many winners have expressed. Here are just a few glorious examples:
+
+Brittany Stewart - [@Brittish_Beauty](https://twitter.com/Brittish_Beauty)
+> Woohoo! I'm extremely honored to be nominated as #TesterOfTheDay! It's a great feeling to know that so many persons have been benefiting from the using this approach. I'm motivated to explore and share more ways of improving efficiency using mind maps. Thank you so much!
+
+Damian Synadinos - [@dsynadinos](https://twitter.com/dsynadinos):
+> Wow! What a pleasant way to start my day!!
+> Thanks to @TestPappy for the nomination & to @FullSnackTester for the recognition!
+> I truly believe that effective communication is essential for well-performing testing teams, and so much more than that...
+> Thank You!
+
+## A very personal winners story
+
+Antonella Scaravilli also known as @RookieSketches on Twitter shared her personal story of being a multi-time winner of #TesterOfTheDay.
+
+> The first time I was nominated I didn't know what it was and I didn't feel like I deserved it at all. However, the second time was quite a surprise and the fact that I had worked so hard on my notes and that someone gave me a recognition for my sketchnoting meant the world to me...and perhaps that's the lesson I learned, that sometimes the contribution to the industry isn't tangible like a book or a course, but that our presence is important because we're all different and we all bring something to the table.
+
+## Hall of Fame
+
+All winners of Tester of The day are inducted into the Testers Hall of Fame.
+
+This years winners can be found on: [Tester of The Day: Hall of Fame](https://testeroftheday.com)
+
+Along with the winners from 2020, who can be found at: [Tester of The Day: Hall of Fame 2020](https://totd-2020.fullsnacktester.com)
+
+## Supported by volunteers
```

Comment:
Word count permitting, could be useful for people to know a little of what the volunteers do